.pipelines/build.yml

```diff
@@ -25,7 +25,21 @@ steps:
   displayName: 'Go lint'
 
 - script: |
-    if go vet -v ./...; then
+    if go vet ./...; then
+        echo -e "\e[42;97m govet SUCCEEDED \e[0m"
```

Comment:
Is it some escaped characters? could we add a comment here?

Reply:
added more `go vet` for the test code. See the -tags fields.